Source/EasyNetQ.Serialization.SystemTextJson/ServiceRegisterExtensions.cs

```diff
@@ -0,0 +1,20 @@
+using EasyNetQ.DI;
+using EasyNetQ.Serialization.SystemTextJson;
+
+// ReSharper disable once CheckNamespace
+namespace EasyNetQ;
+
+/// <summary>
+///     Register serializer based on System.Text.Json
+/// </summary>
+public static class ServiceRegisterExtensions
+{
+    /// <summary>
+    ///     Enables serializer based on System.Text.Json
+    /// </summary>
+    /// <param name="serviceRegister">The register</param>
+    public static IServiceRegister EnableSystemTextJson(this IServiceRegister serviceRegister)
```

Comment:
Maybe better to provide an overload that could configure the `JsonSerializerOptions`

```c#
public static IServiceRegister EnableSystemTextJson(this IServiceRegister serviceRegister, Action<JsonSeriaizerOptions> optionsConfigure)
```